agent/consul/autopilot/autopilot.go

```diff
@@ -215,7 +216,8 @@ func (a *Autopilot) pruneDeadServers() error {
 					a.logger.Printf("[INFO] autopilot: Attempting removal of failed server node %q", member.Name)
 					go serfLAN.RemoveFailedNode(member.Name)
```

Comment:
This is another spot to remove from serfWAN, since non-voting servers are still servers in the WAN pool.